packages/driver/ts/internal-types.d.ts

```diff
@@ -2,13 +2,15 @@
 // TODO: find a better place for this
 
 declare namespace Cypress {
-
   interface Cypress {
+    // TODO: how to pull these from resolvers.ts? can't import in a d.ts file...
+    resolveWindowReference: any
```

Comment:
nice job getting rid of all those `ts-ignore`

Reply:
yeah, as you can see i originally had good intentions of defining these by importing `resolvers.ts` and using `resolveWindowReference: typeof resolveWindowReference`, etc... but ran into an issue. know how to do the TODO?